src/identity/contactMapping.ts

```diff
@@ -239,7 +239,7 @@ export function* fetchWalletAddressesDecentralized(e164Number: string) {
     const accountAddress = normalizeAddressWith0x(accountAddresses[i])
     const walletAddress = normalizeAddressWith0x(address)
     // `getWalletAddress` returns a null address when there isn't a wallet registered
-    if (!isNullAddress(walletAddress)) {
+    if (walletAddress !== NULL_ADDRESS) {
```

Comment:
I was curious how this worked before, and it looks like there's a bug in the [old implementation](https://github.com/celo-org/celo-monorepo/blob/0871aec1c15bb780bb740dda432af725f4bf7c5b/packages/sdk/base/src/address.ts#L9). ugh. They trim the leading 0x, and then compare it to an address with a leading 0x.

Reply:
would've been so simple to catch with unit tests...

Reply:
I opened an issue for it in case they still maintain v1 https://github.com/celo-org/celo-monorepo/issues/10045